### DISCUSSION TITLE
Nuclei engine appears outdated after update

### DISCUSSION THREAD

**Discussion by emadshanab** [projectdiscovery/nuclei#1013] [Q&A] [answered]
**Describe the bug**
After updating the nuclei did not detect the new version and showing Using Nuclei Engine 2.5.1 (outdated)

**Nuclei version**
2.5.1

**Screenshot of the error or bug**
[WRN] Use with caution. You are responsible for your actions.
[WRN] Developers assume no liability and are not responsible for any misuse or damage.
[INF] Supplied input was automatically deduplicated (198 removed).
[WRN] Found 1 templates with syntax warning (use -validate flag for further examination)
[INF] Using Nuclei Engine 2.5.1 (outdated)
[INF] Using Nuclei Templates 8.5.0 (latest)
[INF] Using Interactsh Server https://interact.sh
[INF] Templates added in last update: 0

**Comment by ehsandeep** [comment, marked as answer]
@emadshanab local template config file `~/.config/nuclei/.template-config.yaml` gets updated after 24 hours, so even you are running the latest version, it might be possible that the config file is not updated which eventually will after 24 hours, and outdated flag will be updated, so not an issue.

**Reply by emadshanab** [reply]
Hello @ehsandeep 
Thanks for the fast response. 
Best regards.
